lib/generateMarkdown.js

```diff
@@ -104,14 +108,39 @@ function createPropertiesSummary(schema, suppressWarnings) {
     return md;
 }
 
-function createPropertiesDetails(schema, title, headerLevel, suppressWarnings) {
-    var headerMd = style.getHeaderMarkdown(headerLevel);
+function createPatternPropertiesSummary(schema, suppressWarnings) {
```

Comment:
I'm trying to improve the code documentation as I go, making sure that I add [JSHint](http://jshint.com/) documentation to any new function I add, or old function that I update significantly.  It would be great if you could do so as well.  I've been using the [lit-it](https://marketplace.visualstudio.com/items?itemName=mohseenrm.lit-it) VSCode extension to make this easier.